### ISSUE TITLE
transformer_asr.py loss is so big,it doesnt work as the comment say.I make any mistake?

### ISSUE THREAD

**Issue by doodoocoder** [keras-team/keras-io#598]
I just modify the epochs to 100 as the comment say ,like this:

history = model.fit(ds, validation_data=val_ds, callbacks=[display_cb], epochs=100)

but after 100 epochs ,the part of result is this:

```python
Epoch 87/100
203/203 [==============================] - 188s 925ms/step - loss: 0.9083 - val_loss: 0.9267
Epoch 88/100
203/203 [==============================] - 196s 961ms/step - loss: 0.9076 - val_loss: 0.9267
Epoch 89/100
203/203 [==============================] - 203s 996ms/step - loss: 0.9070 - val_loss: 0.9268
Epoch 90/100
203/203 [==============================] - 192s 944ms/step - loss: 0.9064 - val_loss: 0.9270
Epoch 91/100
203/203 [==============================] - 196s 962ms/step - loss: 0.9058 - val_loss: 0.9271
target:     <the increased information supplied by other agencies will be wasted.>
prediction: <the commission of the president service and the president service and the president and the president and the presidential and secret secreased the responsion ten.>

target:     <prs must develop the capacity to classify its subjects on a more sophisticated basis than the present geographic breakdown.>
prediction: <the commission of the president service and the president service and the president and the president and the presidential and secret secreased the responsion ten.>

target:     <its present manual filing system is obsolete#>
prediction: <the commission of the president service and the president service and the president and the president and the presidential and secret secreased the responsion ten.>

target:     <it makes no use of the recent developments in automatic data processing which are widely used in the business world and in other government offices.>
prediction: <the commission of the president service and the president service and the president and the president and the presidential and secret secreased the responsion ten.>

Epoch 92/100
203/203 [==============================] - 193s 947ms/step - loss: 0.9053 - val_loss: 0.9271
Epoch 93/100
203/203 [==============================] - 193s 948ms/step - loss: 0.9047 - val_loss: 0.9273
Epoch 94/100
203/203 [==============================] - 192s 943ms/step - loss: 0.9042 - val_loss: 0.9276
Epoch 95/100
203/203 [==============================] - 193s 946ms/step - loss: 0.9036 - val_loss: 0.9278
Epoch 96/100
203/203 [==============================] - 194s 955ms/step - loss: 0.9030 - val_loss: 0.9281
target:     <the increased information supplied by other agencies will be wasted.>
prediction: <the commission of the secret service and the secret service and the president service and the president of the presidential and the secreation the assassinathe thenteren>

target:     <prs must develop the capacity to classify its subjects on a more sophisticated basis than the present geographic breakdown.>
prediction: <the commission of the secret service and the secret service and the president service and the president of the presidential and the secreation the assassinathe thenteren>

target:     <its present manual filing system is obsolete#>
prediction: <the commission of the secret service and the secret service and the president service and the president of the presidential and the secreation the assassinathe thenteren>

target:     <it makes no use of the recent developments in automatic data processing which are widely used in the business world and in other government offices.>
prediction: <the commission of the secret service and the secret service and the president service and the president of the presidential and the secreation the assassinathe thenteren>

Epoch 97/100
203/203 [==============================] - 195s 959ms/step - loss: 0.9025 - val_loss: 0.9282
Epoch 98/100
203/203 [==============================] - 197s 968ms/step - loss: 0.9019 - val_loss: 0.9286
Epoch 99/100
203/203 [==============================] - 197s 966ms/step - loss: 0.9013 - val_loss: 0.9293
Epoch 100/100
203/203 [==============================] - 194s 953ms/step - loss: 0.9006 - val_loss: 0.9300
```

who can tell me where the question is?

**Comment by sachinprasadhs** [comment]
Could you please clarify on the issue here.

If you're specific about the loss, there is no upper bound for loss in [Categorical CrossEntropy](https://keras.io/api/losses/probabilistic_losses/#categoricalcrossentropy-class) it ranges from `[0, inf] `
